src/main/java/racingcar/model/domain/Racing.java

```diff
@@ -0,0 +1,25 @@
+package racingcar.model.domain;
+
+public class Racing {
+
+    private Cars cars;
+    private CarMoveBehavior carMoveBehavior;
+
+    public Racing(Cars cars, final CarMoveBehavior carMoveBehavior) {
+        this.cars = cars;
+        this.carMoveBehavior = carMoveBehavior;
+    }
+
+    public void drive() {
+
+        cars.cars().stream()
```

Comment:
* 여기도 getter가 존재하는 느낌인데요! 해당 부분도 Cars객체에게 메세지를 전달하는  방식으로 구현하면 어떨까요?
* `stream` 없이 forEach가 가능해요! [참고](https://dundung.tistory.com/247)

```suggestion
        cars.cars().forEach(car -> racingCondition(car));
```

Reply:
ㅎㅎ 반영했습니다!!